run_build.sh

```diff
@@ -18,15 +51,24 @@ git submodule init
 git submodule update
 git merge --no-edit origin/tanabarr/control-no-ipmctl-May2020
 git merge --no-edit origin/mjmac/allow-fwd-disable-20200508
-scons --build-deps=yes --config=force install
+scons MPI_PKG=any --build-deps=yes --config=force install
 popd
 popd
 popd
 
+
+
 pushd $IOR_DIR
 ./bootstrap
-./configure --prefix=$BUILD_DIR/latest/daos/install/ior_openmpi --with-daos=$BUILD_DIR/latest/daos/install --with-cart=$BUILD_DIR/latest/daos/install MPICC=$MPI_DIR/bin/mpicc CPPFLAGS="-I$BUILD_DIR/latest/daos/install/prereq/dev/mercury/include -I$MPI_DIR/include" LDFLAGS=-L$MPI_DIR/lib LIBS=-lmpi
+./configure --prefix=${LATEST_DAOS}/ior${MPI_SUFFIX} \
+            --with-daos=${LATEST_DAOS} \
+            --with-cart=${LATEST_DAOS} \
+            CPPFLAGS=-I${LATEST_DAOS}/prereq/dev/mercury/include \
+            LIBS=-lmpi
 make clean
 make
```

Comment:
Similarly, can we check for make success?

Reply:
The new trap will check the return code